src/twin/system_info.rs

```diff
@@ -208,18 +234,26 @@ impl SystemInfo {
         };
 
         tx.send(json!({
-            "system_info": serde_json::to_value(&self.info).context("report: cannot serialize")?
+            "system_info": serde_json::to_value(&self.software_info).context("report: cannot serialize")?
         }))
         .await
         .context("report: send")
     }
 
+    fn boot_time() -> Result<String> {
+        let boot_time = time::OffsetDateTime::UNIX_EPOCH
+            + std::time::Duration::from_secs(sysinfo::System::boot_time());
```

Comment:
is `sysinfo::System::boot_time()` boot_time or uptime? If it is boot_time we don't need to calculate it anymore and can use it directly...